src/prettytable/prettytable.py

```diff
@@ -1887,7 +1887,10 @@ def _stringify_title(self, title, options):
         )
         bits.append(endpoint)
         title = " " * lpad + title + " " * rpad
-        bits.append(self._justify(title, len(self._hrule) - 2, "c"))
+        lpad, rpad = self._get_padding_widths(options)
+        sum_widths = sum(n + lpad + rpad + 1 for n in self._widths)
```

Comment:
Actually, my last suggestion was wrong, and summing a list comprehension is faster than a generator:

```console
❯ python -m timeit -s "a = (1,2,3,4,4,6,7,8,9)" "sum(tuple(n for n in a))"
1000000 loops, best of 5: 363 nsec per loop

❯ python -m timeit -s "a = (1,2,3,4,4,6,7,8,9)" "sum(n for n in a)"
1000000 loops, best of 5: 270 nsec per loop

❯ python -m timeit -s "a = (1,2,3,4,4,6,7,8,9)" "sum([n for n in a])"
2000000 loops, best of 5: 168 nsec per loop
```


```suggestion
        sum_widths = sum([n + lpad + rpad + 1 for n in self._widths])
```